neural_lam/models/hi_lam_parallel.py

```diff
@@ -2,18 +2,17 @@
 import torch
 import torch_geometric as pyg
 
-# First-party
-from neural_lam.interaction_net import InteractionNet
-from neural_lam.models.base_hi_graph_model import BaseHiGraphModel
+# Local
+from ..interaction_net import InteractionNet
+from .base_hi_graph_model import BaseHiGraphModel
 
 
 class HiLAMParallel(BaseHiGraphModel):
-    """
-    Version of HiLAM where all message passing in the hierarchical mesh (up,
-    down, inter-level) is ran in parallel.
+    """Version of HiLAM where all message passing in the hierarchical mesh (up, down,
+    inter-level) is ran in parallel.
+
+    This is a somewhat simpler alternative to the sequential message passing of Hi-LAM.
 
-    This is a somewhat simpler alternative to the sequential message passing
-    of Hi-LAM.
     """
 
     def __init__(self, args):
```

Comment:
Here the following two args are missing
```suggestion
def __init__(self, args, datastore, forcing_window_size):
        super().__init__(args, datastore, forcing_window_size)
```

Reply:
fixed in https://github.com/leifdenby/neural-lam/commit/1889771efcc04ce304988e54045950fe4cc46e29